docs/UserGuide.md

```diff
@@ -147,34 +147,43 @@ Borrow a book from the library by a user.
 
 Format: `borrow INDEX BOOKTITLE`
 
-* Borrow the book with `BOOKTITLE` to user `INDEX`
+* Borrow the book with `BOOKTITLE` to user `INDEX`.
 * The index refers to the index number shown in the displayed User List.
 * The index **must be a positive integer** 1, 2, 3, …​
 
 Examples:
-* `borrow 2 I Love CS2103T CS2101` will record user index 2, borrows a book called "I Love CS2103T CS2101".
-* `borrow 4 The Hero with a Thousand Faces` will record user index 4, borrows a book called "The Hero with a Thousand Face".
+* `borrow 2 b/I Love CS2103T CS2101` will record user index 2, borrows a book called "I Love CS2103T CS2101".
+* `borrow 4 b/The Hero with a Thousand Faces` will record user index 4, borrows a book called "The Hero with a Thousand Face".
 
 ### Return a book from a user : `return`
 
 Return the book borrowed by a user.
 
-Format: `return INDEX`
+Format: `return INDEX BOOKTITLE`
 
-* Return the book borrowed by user `INDEX`
+* Return the book borrowed by user `INDEX` and `BOOKTITLE`.
```

Comment:
I feel like it would sounds more understandable if we change it into
Return the book `BOOKTITLE` borrowed by user `INDEX`.